src/diffuse/utils.py

```diff
@@ -182,14 +182,13 @@ def popenRead(dn, cmd, prefs, bash_pref, success_results=None):
         info = None
     if _use_flatpak():
         cmd = ['flatpak-spawn', '--host'] + cmd
-    with (subprocess.Popen(
+    with subprocess.Popen(
         cmd,
         stdin=subprocess.PIPE,
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
         cwd=dn,
-        startupinfo=info) as proc
-    ):
+        startupinfo=info) as proc:
```

Comment:
This will fail in Flake8.

Could you change it with an additional indentation for the params in order to easily differentiate the with statement and its body?
```python
    with subprocess.Popen(
            cmd,
            stdin=subprocess.PIPE,
            stdout=subprocess.PIPE,
            stderr=subprocess.PIPE,
            cwd=dn,
            startupinfo=info) as proc:
        proc.stdin.close()
```

Reply:
Done